### DISCUSSION TITLE
Bug Report: i installed all..

### DISCUSSION THREAD

**Discussion by boyangsic** [FunkinCrew/Funkin#1682] [Compiling help]
i installed with .vsconfig file. but its still have 'windows.h': No such file or directory or 'crtdbg.h': No such file or directory error. how to fix this?

**Comment by libauror-a** [comment]
Did you install MSVC and the Windows SDK?

**Comment by boyangsic** [comment]
if .vsconfig file have all, i installed all to build.

**Comment by boyangsic** [comment]
ok. so.. i copied file from working computer but it have same error.

**Comment by libauror-a** [comment]
think thats because you need to make env vars
